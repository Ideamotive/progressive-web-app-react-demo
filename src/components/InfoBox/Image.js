import React from 'react';
import styled from 'styled-components';

export default styled.div`
    width: 100%;
    height: 300px;
    background-size: cover;
    background-image: ${props => props.backgroundImage};
    `;