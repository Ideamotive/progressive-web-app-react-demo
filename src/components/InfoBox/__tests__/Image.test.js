import React from 'react';
import * as renderer from 'react-test-renderer';

import Image from '../Image';

describe('<Image/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(
        <Image
          backgroundImage={'https://test.test'}/>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

