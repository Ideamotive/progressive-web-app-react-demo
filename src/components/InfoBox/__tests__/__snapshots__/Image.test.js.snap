// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa czEITg"
/>
`;
